serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/servicerecord/ServiceRecordFactory.java

```diff
@@ -91,7 +103,7 @@ public ServiceRecordFactory(ServiceregistryProperties properties,
         try {
             pemCertificate = lookupPemCertificate(orgnr);
         } catch (CertificateNotFoundException e) {
-            logger.info(MarkerFactory.receiverMarker(orgnr), String.format("Identifier %s found in ELMA with " +
+            log.info(MarkerFactory.receiverMarker(orgnr), String.format("Identifier %s found in ELMA with " +
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Identifier %s found in ELMA with " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1192)